src/Test.ConcurrencyTests/BasicCoyoteTests.cs

```diff
@@ -36,6 +36,7 @@ public async Task TestTaskAsync()
         }
 
         [TestMethod, TestCategory("NightlyTest")]
+        [Ignore("Skipping all tests in this class temporarily.")]
```

Comment:
I assume we're skipping these since the results are nondeterministic right now, what's our plan long term to tackle these? Additionally - can you please extract this change into a separate commit, it's rather hidden in a bunch of other changes right now.